dff/messengers/telegram/abstract.py

```diff
@@ -0,0 +1,364 @@
+                                caption=attachment.__pydantic_extra__.get("caption", None),
+                                parse_mode=attachment.__pydantic_extra__.get("parse_mode", None),
+                                has_spoiler=attachment.__pydantic_extra__.get("has_spoiler", None),
+                                thumbnail=attachment.__pydantic_extra__.get("thumbnail", None),
+                            ),
+                        ]
+                if isinstance(attachment, Image):
+                    attachment_bytes = await attachment.get_bytes(self)
+                    if attachment_bytes is not None:
+                        files += [
+                            InputMediaPhoto(
+                                attachment_bytes,
+                                filename=attachment.__pydantic_extra__.get("filename", None),
+                                caption=attachment.__pydantic_extra__.get("caption", None),
+                                parse_mode=attachment.__pydantic_extra__.get("parse_mode", None),
+                                has_spoiler=attachment.__pydantic_extra__.get("has_spoiler", None),
+                            ),
+                        ]
+                if isinstance(attachment, Document):
+                    attachment_bytes = await attachment.get_bytes(self)
+                    if attachment_bytes is not None:
+                        files += [
+                            InputMediaDocument(
+                                attachment_bytes,
+                                filename=attachment.__pydantic_extra__.get("filename", None),
+                                caption=attachment.__pydantic_extra__.get("caption", None),
+                                parse_mode=attachment.__pydantic_extra__.get("parse_mode", None),
+                                disable_content_type_detection=attachment.__pydantic_extra__.get(
+                                    "disable_content_type_detection", None
+                                ),
+                                thumbnail=attachment.__pydantic_extra__.get("thumbnail", None),
+                            ),
+                        ]
+                if isinstance(attachment, Keyboard):
+                    buttons = attachment.buttons
+            if len(files) > 0:
+                await bot.send_media_group(
+                    chat_id,
+                    files,
+                    disable_notification=message.__pydantic_extra__.get("disable_notification", None),
+                    protect_content=message.__pydantic_extra__.get("protect_content", None),
+                    caption=message.text,
+                )
+                return
+        if message.text is not None:
+            await bot.send_message(
+                chat_id,
+                message.text,
+                parse_mode=attachment.__pydantic_extra__.get("parse_mode", None),
```

Comment:
`attachment` may not be defined here.
I feel like I cannot merge this PR without adding tests first.
So could you add telegram tests in this PR instead of a separate one?